src/commands/variables.rs

```diff
@@ -73,6 +98,30 @@ impl VariablesCommand {
                         .await?;
                 set_variables(&info.client, info.app_id, &cmd.variables_to_set).await?;
             }
+            Self::Delete(cmd) => {
+                let info =
+                    AppManagmentInfo::new(cmd.common.deployment_env_id.as_deref(), &cmd.common.app)
+                        .await?;
+                delete_variables(&info.client, info.app_id, &cmd.variables_to_delete).await?;
+            }
+            Self::List(cmd) => {
+                let info =
+                    AppManagmentInfo::new(cmd.common.deployment_env_id.as_deref(), &cmd.common.app)
+                        .await?;
+                let vars = get_variables(&info.client, info.app_id).await?;
+                let var_names = vars
+                    .iter()
+                    .map(|var| from_str(var))
+                    .collect::<Result<Vec<Variable>, _>>()
+                    .context("could not parse variable")?;
```

Comment:
For consistency with other contexts here:

```suggestion
                    .context("Could not parse variable")?;
```